client/src/main/java/io/split/client/SplitFactoryBuilder.java

```diff
@@ -39,6 +39,9 @@ public static SplitFactory build(String apiToken) throws IOException, URISyntaxE
     public static synchronized SplitFactory build(String apiToken, SplitClientConfig config) throws IOException, URISyntaxException {
         ApiKeyValidator.validate(apiToken);
         if (LocalhostSplitFactory.LOCALHOST.equals(apiToken)) {
+            if (config.splitFile().endsWith("json")){
```

Comment:
I think that you should cover `.json` and also put everything in lowercase just in case

Reply:
Now we check that the file could be json, JSON or Json.